Addon will not stay running

# Problem/Motivation

When trying to access the webui, I get a popup stating **Add-on is not running. Please start it first**
When I check the add-on it is indeed stopped. Clicking on start seems to make it start (green dot, webui button appears) but clicking on web ui gets the pop up again and the add-on is stopped

I have tried restarting and reinstalling

Logs seem to indicate its running

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service base-addon-banner: starting
-----------------------------------------------------------
 Add-on: Studio Code Server
 Fully featured Visual Studio Code (VSCode) experience integrated in the Home Assistant frontend.
-----------------------------------------------------------
 Add-on version: 5.8.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 10.2  (aarch64 / odroid-n2)
 Home Assistant Core: 2023.6.2
 Home Assistant Supervisor: 2023.06.4
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-timezone: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
[09:03:33] INFO: Configuring timezone (Europe/London)...
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service base-addon-timezone successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-user: starting
s6-rc: info: service init-mysql: starting
s6-rc: info: service init-mosquitto: starting
s6-rc: info: service init-code-server: starting
s6-rc: info: service init-code-server successfully started
s6-rc: info: service init-mysql successfully started
s6-rc: info: service init-user successfully started
s6-rc: info: service code-server: starting
s6-rc: info: service code-server successfully started
[09:03:34] INFO: Starting code-server...
s6-rc: info: service init-mosquitto successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
[2023-06-28T08:03:36.202Z] info  Wrote default config file to ~/.config/code-server/config.yaml
[2023-06-28T08:03:37.161Z] info  code-server 4.14.1 5c199629305a0b935b4388b7db549f77eae82b5a
[2023-06-28T08:03:37.163Z] info  Using user-data-dir /data/vscode
[2023-06-28T08:03:37.206Z] info  Using config file ~/.config/code-server/config.yaml
[2023-06-28T08:03:37.206Z] info  HTTP server listening on http://0.0.0.0:1337/
[2023-06-28T08:03:37.207Z] info    - Authentication is disabled
[2023-06-28T08:03:37.207Z] info    - Not serving HTTPS
[2023-06-28T08:03:37.207Z] info  Session server listening on /data/vscode/code-server-ipc.sock
[09:05:08] 
[09:05:08] Extension host agent started.
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda_bg.wasm
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda.js
[09:05:09] [172.30.32.2][76debc08][ManagementConnection] New connection established.
[09:05:09] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: Cannot find module '../build/Release/keytar.node'
Require stack:
- /usr/local/lib/code-server/lib/vscode/node_modules/keytar/lib/keytar.js
- /usr/local/lib/code-server/lib/vscode/out/bootstrap-amd.js
- /usr/local/lib/code-server/out/node/util.js
- /usr/local/lib/code-server/out/node/cli.js
- /usr/local/lib/code-server/out/node/entry.js
[09:05:10] [172.30.32.2][7a6876b6][ExtensionHostConnection] New connection established.
[09:05:12] [172.30.32.2][7a6876b6][ExtensionHostConnection] <568> Launched Extension Host Process.
[09:05:20] [172.30.32.2][76debc08][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[09:05:20] [172.30.32.2][7a6876b6][ExtensionHostConnection] <568> Extension Host Process exited with code: 0, signal: null.
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda_bg.wasm
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda.js
[09:05:49] [172.30.32.2][fe0b8dd8][ManagementConnection] New connection established.
[09:05:50] [172.30.32.2][015b725d][ExtensionHostConnection] New connection established.
[09:05:50] [172.30.32.2][015b725d][ExtensionHostConnection] <652> Launched Extension Host Process.
[09:06:19] [172.30.32.2][fe0b8dd8][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[09:06:19] [172.30.32.2][015b725d][ExtensionHostConnection] <652> Extension Host Process exited with code: 0, signal: null.
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda_bg.wasm
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda.js
[09:06:21] [172.30.32.2][e13b5298][ManagementConnection] New connection established.
[09:06:22] [172.30.32.2][8a09b850][ExtensionHostConnection] New connection established.
[09:06:22] [172.30.32.2][8a09b850][ExtensionHostConnection] <814> Launched Extension Host Process.
[09:06:34] [172.30.32.2][e13b5298][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[09:06:34] [172.30.32.2][8a09b850][ExtensionHostConnection] <814> Extension Host Process exited with code: 0, signal: null.
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda_bg.wasm
File not found: /usr/local/lib/code-server/lib/vscode/out/vsda.js
[09:06:36] [172.30.32.2][23eba89e][ManagementConnection] New connection established.
[09:06:36] [172.30.32.2][99d32f03][ExtensionHostConnection] New connection established.
[09:06:36] [172.30.32.2][99d32f03][ExtensionHostConnection] <922> Launched Extension Host Process.
```

## Expected behavior

VSCode accessible when clicking on webui button (or the sidebar button)

## Actual behavior

As above

## Steps to reproduce

Click on webui button (or the sidebar button)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

--- a/vscode/Dockerfile
+++ b/vscode/Dockerfile
@@ -131,9 +131,7 @@ COPY rootfs /
 
 # Health check
 HEALTHCHECK \
-    CMD curl --fail http://127.0.0.1:1337/healthz \
-        | jq --exit-status '.status == "alive"' \
-        || exit 1
+    CMD curl --fail http://127.0.0.1:1337/healthz || exit 1
 
 # Build arguments
 ARG BUILD_ARCH

